app/controlplane/internal/service/workflowcontract.go

```diff
@@ -109,10 +118,16 @@ func (s *WorkflowContractService) Update(ctx context.Context, req *pb.WorkflowCo
 		return nil, err
 	}
 
+	token, err := usercontext.GetRawToken(ctx)
+	if err != nil {
+		return nil, status.Error(codes.Unauthenticated, "invalid token")
+	}
+
 	schemaWithVersion, err := s.contractUseCase.Update(ctx, currentOrg.ID, req.Name,
```

Comment:
are you passing the token on update?

Reply:
Yes, it's being passed both on create and update.